openapi-cli/src/test/resources/expected_gen/generic_service_petstore.bal

```diff
@@ -0,0 +1,12 @@
+import ballerina/http;
+
+listener http:Listener ep0 = new (80, config = {host: "petstore.openapi.io"});
+
+service /v1 on ep0 {
+    resource function get pets(http:Caller caller, http:Request request) {
+    }
+    resource function post pets(http:Caller caller, http:Request request) {
+    }
+    resource function get pets/[string petId](http:Caller caller, http:Request request) {
```

Comment:
Will it add a function body statement for the complex path `/admin/api/2021-10/customers/{customer_id}.json`?